test/logging.cc

```diff
@@ -57,12 +57,12 @@ TEST(Logging, FileLoggerTest) {
     results.emplace_back(std::async(std::launch::async, work));
   }
 
-  // dont really care about the results but we can pretend
-  int exit_code = 0;
   for (auto& result : results) {
     try {
       result.get();
```

Comment:
instead of doing try catch at all could you just do this:

```suggestion
      ASSERT_NO_THROW(result.get());
```

Reply:
done.